Code/GraphMol/DistGeomHelpers/catch_tests.cpp

```diff
@@ -1190,3 +1190,47 @@ TEST_CASE("allenes and cumulenes") {
     }
   }
 }
+
+namespace RDKit {
+namespace DGeomHelpers {
+namespace EmbeddingOps {
+void findDoubleBonds(
```

Comment:
I can't reproduce the CI problems when doing a DLL build on my windows machine in the usual way, but it's worth trying this change to see if it helps:
```suggestion
RDKIT_DISTGEOMHELPERS_EXPORT void findDoubleBonds(
```
If that doesn't do it, I will dig a bit more